pages/en/lb4/Installation.md

```diff
@@ -12,111 +12,26 @@ summary:
 
 Install the following:
 
-- [Node.js](https://nodejs.org/en/download/) version 7 or higher.
+- Node.js, either:
+  - [Version 6.x](https://nodejs.org/en/download/): The long-term support (LTS) version.
+  - **Recommended**: [Version 8.x](https://nodejs.org/en/download/current/): The latest/current version.
```

Comment:
Can we reverse the order, start with the recommended version 8.x, and then mention that 6.x can be used too?